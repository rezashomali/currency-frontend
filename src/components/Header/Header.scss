.header {
  .header__logo {
    margin-right: 20px;
  }
}
