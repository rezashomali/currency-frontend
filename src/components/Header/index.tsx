import React from "react";
import { AppBar, Toolbar, Typography, Grid } from "@material-ui/core";
import "./Header.scss";

const Header = () => {
  return (
    <div className="header">
      <AppBar position="static">
        <Toolbar className="">
          <Typography className="header__logo" variant="h2" color="inherit">
            Klarx
          </Typography>
          <Typography variant="h6"> Currency Converter</Typography>
        </Toolbar>
      </AppBar>
    </div>
  );
};

export default Header;
