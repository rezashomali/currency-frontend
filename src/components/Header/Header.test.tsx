import React from "react";
import renderer from "react-test-renderer";
import Header from "./index";

test("Header snapshot ", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
