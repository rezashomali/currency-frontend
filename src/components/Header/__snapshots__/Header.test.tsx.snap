// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot  1`] = `
<div
  className="header"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary header__appBar MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h2
        className="MuiTypography-root header__logo MuiTypography-h2 MuiTypography-colorInherit"
      >
        Klarx
      </h2>
    </div>
  </header>
</div>
`;
